MvvmCross/Core/MvxSetup.cs

```diff
@@ -54,7 +57,10 @@ public static IMvxSetup Instance()
 
         protected abstract IMvxViewsContainer CreateViewsContainer();
 
-        protected abstract IMvxViewDispatcher CreateViewDispatcher();
+        protected virtual IMvxViewDispatcher CreateViewDispatcher()
+        {
+            return Mvx.Resolve<IMvxViewDispatcher>();
```

Comment:
There must be a reason for doing / returning just `Mvx.Resolve<IMvxViewDispatcher>` inside a method that claims to create a ViewDispatcher, right? Would be nice to clarify this somehow

Reply:
Good point - worth having some comments/docs around this. The change is that the implementation is lazy registered, which means that calling resolve here will create the instance. This method is overridable to allow for developers to override the creation of the view dispatcher. In reality the best way to override the view presenter is to override the initializeIoC and register a different implementaiton